lib/rbs/types.rb

```diff
@@ -1212,7 +1212,57 @@ def to_json(state = _ = nil)
       end
 
       def to_s(level = 0)
-        literal.inspect
+        case literal
+        when String
+          string_to_s(literal)
+        when Symbol
+          symbol_to_s(literal)
+        else
+          literal.inspect
+        end
+      end
+
+      private
+
+      def string_to_s(str)
+        content = str.chars.map do |c|
+          case c
```

Comment:
Wouldnt a hash and `[]` work better? like `str.chars.map { |c| SPECIAL_CHARS.fetch(c, c) }` ?